utils/config.js

```diff
@@ -9,6 +9,7 @@ const config = {
     dev: 'https://safe-team.dev.gnosisdev.com/app/',
```

Comment:
We can remove Rinkeby environment, as it's deprecated

Reply:
In `e2e-run-on-main.yml`I can see a `TESTING_ENV: 'rinkeby'`

Changing to `stg`?

Reply:
I would say:

TESTING_ENV: 'prod'
NETWORK_NAME: 'rinkeby'
NETWORK_ADDRESS_PREFIX: 'rin'

For now it will execute the same test

Reply:
read_only tests working locally @dasanra 👻

Problem was the paths to static files (as the QR or CSV) as I created separated folders for web and apps tests and vscode cannot automatically detect paths outside imports. So forget to change those ones

Sending the task to QA